packages/ui/src/components/shared/export/QrCode.tsx

```diff
@@ -39,17 +31,27 @@ export default function QrCode({ walletBackup }: QrCodeProps) {
         return;
       }
 
-      FileSaver.saveAs(blob, `coongwallet_backup_qrcode_${Date.now()}`);
+      FileSaver.saveAs(
+        blob,
+        `coongwallet_${object.toLowerCase()}_backup_qrcode_${detail?.toLowerCase().replace(/\s/g, '')}_${Date.now()}`,
```

Comment:
I think we can extract the account name from the value rather than passing a `detail` prop here which might be a bit confusing here.
1. When when `detail` is empty (in the case object = `Wallet`), the file name might have 2 consecutive underscores `__` since `${detail?.toLowerCase().replace(/\s/g, '')}` will be evaluated to an empty string.
2. Instead of using `detail` prop, we can create a function to generate qr code file name for each `object` & `value` case, something like below:

```ts
const getFileName = () => {
  if (value === ExportObject.Wallet) {
    return `coongwallet_wallet_backup_${Date.now()}.json`    
  } else if (value === ExportObject.Account) {
    const accountName = (value?.meta?.name || '').toLowerCase().replace(/\s/g, '_')
    return `coongwallet_account_backup_${accountName}_${Date.now()}.json`    
  }
}
```

We can do this for the `Json` component as well.